service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingService.kt

```diff
@@ -153,7 +154,7 @@ interface RevurderingService {
     fun avsluttRevurdering(
         revurderingId: UUID,
         begrunnelse: String,
-        fritekst: String?,
+        brevvalg: Brevvalg.SaksbehandlersValg?,
```

Comment:
Litt enklere å endre/utvide på sikt dersom vi ikke er alt for spesifikke?

Reply:
Jeg ønsket ikke å ha den for åpen i servicelaget, og heller la domenet styre hva som er lov. Kunne evt. byttet ut null med en ekstra type som tilsvarer at saksbehandler ikke har et valg i dette tilfellet.